instance/models/server.py

```diff
@@ -287,8 +289,8 @@ def public_ip(self):
 
         try:
             public_addr = openstack.get_server_public_address(self.os_server)
-        except novaclient.exceptions.NotFound:
-            return None  # This server has been deleted.
+        except Exception:  # pylint: disable=broad-except
```

Comment:
No problem! It's a small world, I was checking out edX/OpenCraft, and noticed you worked on it. Anyways, don't mean to disrupt the review. Cheers 🍻